frontend/pages/discover/project-developers.tsx

```diff
@@ -68,7 +68,7 @@ const ProjectDevelopersPage: PageComponent<ProjectDevelopersPageProps, DiscoverP
                 <Loading visible={loading} iconClassName="w-10 h-10" />
               </span>
             )}
-            <div className="grid grid-cols-1 gap-6 p-1 md:grid-cols-2 xl:grid-cols-3">
+            <div className="grid grid-cols-1 gap-2 sm:p-1 sm:gap-6 md:grid-cols-2 xl:grid-cols-3">
```

Comment:
For consistency with all the other card types, let's set the same padding:
```suggestion
            <div className="grid grid-cols-1 gap-2 p-0.5 sm:gap-6 md:grid-cols-2 xl:grid-cols-3">
```